### DISCUSSION TITLE
Cannot find the correct path when checkout the project into a specific folder

### DISCUSSION THREAD

**Discussion by fatihtokus** [JamesIves/github-pages-deploy-action#1412] [Q&A]
- name: Checkout
  uses: actions/checkout@v3
  with:
    path: main_project

 - name: Deploy 🚀
        uses: JamesIves/github-pages-deploy-action@v4
        with:
          folder: scan-results # This folder exists 

Then I got the following error:
Checking configuration and starting deployment… 🚦
Error: The directory you're trying to deploy named /home/runner/work/infra-audit-test/infra-audit-test/scan-results doesn't exist. Please double check the path and any prerequisite build scripts and try again. ❗

But It should be looking at the  /home/runner/work/main_project/infra-audit-test/infra-audit-test/scan-results

**Comment by JamesIves** [comment]
Are you sure that the `scan-results` folder actually exists in this context? You might want to try running `ls` between checkout and deploy to see where the folder is located, it may be nested somewhere.
